data/data/agent/files/usr/local/bin/get-interactive-config.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+function connectivity_check() {
+	declare target
+
+	target="$1"
+
+	(>&2 echo "Pinging ${target} to check the validity of the current network configuration")
+	if ping -c 4 "$NODE_ZERO_IP"; then
+		(>&2 echo "Successfully pinged ${target}.")
+		return 0
+	else
+		(>&2 echo "Failed to ping ${target}.")
+		return 1
+	fi
+}
+
+function print_addressing_info() {
+	declare with_prefix
+
+	with_prefix="${1:-true}"
+	echo "Currently configured addresses"
+	if $with_prefix; then
+		ip -j a show  | jq -r '. | map({name: .ifname, addresses: [.addr_info[] | .local + "/" + (.prefixlen | tostring)]}) | .[] | select(.addresses | length > 0) | select( .name != "lo") | .name + ": " + (.addresses | join(" "))'
+	else
+		ip -j a show  | jq -r '. | map({name: .ifname, addresses: [.addr_info[] | .local]}) | .[] | select(.addresses | length > 0) | select( .name != "lo") | .name + ": " + (.addresses | join(" "))'
+	fi
+}
+
+NODE_ZERO=false
+while true; do
+	read -e -p "Do you wish for this node to be the one that runs the installation service (Only one node may perform this function)? (Y)es/(N)o: " NODE0_ANSWER
+	if [[ "$NODE0_ANSWER" =~ ^[Yy](es)?$ ]]; then
+		NODE_ZERO=true
+		break
+	elif [[ "$NODE0_ANSWER" =~ ^[Nn](o)?$ ]]; then
+		(>&2 echo "This node will not run the installation service.")
+		break
+	else
+		(>&2 echo "Invalid answer.")
+	fi
+done
+
+if $NODE_ZERO; then
+	print_addressing_info
```

Comment:
The print_addressing_info is only displayed for Node0. The other nodes don't show their currently configured ip address. So it is unclear whether the nmtui should be used.